### PR TITLE
Add first update functions to client

### PR DESCRIPTION
**Pull request by singularo**
Make deploymentconfigs return a single one if the name is given
Fix some missing leading '/'
Add update functions for deploymentconfig and replicationcontrollers
Recurse to process multiples if a label is passed in

### CODE REVIEW

**Comment by bart88** [comment]
Changes in the routing will need to be checked against #6

**Comment by bart88** [comment]
Happy for this to be merged.

File: src/Client.php [hunk 1]
```diff
@@ -967,100 +971,18 @@ public function createDeploymentConfig(string $name, string $image_stream_tag, s
   /**
    * {@inheritdoc}
    */
-  public function updateDeploymentConfig(string $name, string $image_stream_tag, string $image_name, array $volumes, array $data) {
+  public function updateDeploymentConfig(string $name, int $replica_count) {
```
Comment (caseyfw) [inline src/Client.php]: Wait, so now the only thing you can edit is the replica count?

**Comment by singularo** [comment]
Removed extra whitespace changes I had in there
The labels need the name there to do the 'grouping' of services in openshift.
Tested with develop, @bart88 stuff still works.

**Pull request merged**